.github/workflows/kani.yml

```diff
@@ -35,4 +35,11 @@ jobs:
           submodules: true
 
       - name: Run Kani Script
```

Comment:
Should this be named "Test Kani Script?"

Reply:
I think this first step is in fact checking the standard library with Kani, right?

Reply:
Yes, i shall separate the steps into test kani, following which I shall run kani on the standard library on the whole.